specification/quantum/resource-manager/Microsoft.Quantum/preview/2019-11-04-preview/examples/quantumWorkspacesGet.json

```diff
@@ -0,0 +1,56 @@
+{
+  "parameters": {
+    "workspaceName": "quantumworkspace1",
+    "resourceGroupName": "quantumResourcegroup",
+    "subscriptionId": "1C4B2828-7D49-494F-933D-061373BE28C2",
+    "api-version": "2019-11-04-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "quantumworkspace1",
```

Comment:
>name [](start = 9, length = 4)

GET response must include `id`, `type`, and `name` properties.

Reply:
This comment applies to all other examples as well. Any resource body response should have these properties.

---
In reply to: [553671951](https://github.com/Azure/azure-rest-api-specs/pull/11266#discussion_r553671951) [](ancestors = 553671951)

Reply:
Looks like this was addressed by Andres at commit 7269aa8 already.